File: src/model/Record.kt
package moe.msm.model

import com.fasterxml.jackson.annotation.JsonIgnore
import java.time.LocalDateTime

/**
 * @author sam_nya (privateamusement@protonmail.com)
 */
class Record(
    val id: Long,
    @JsonIgnore
    val network: Network,
    val latency: Long?,
    val isUp: Boolean?,
    val time: LocalDateTime
)

File: Linkmon-desktop/model/Record.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Linkmon_desktop.model
{
    public class Record
    {
        public long Id { get; set; }
        public long Latency { get; set; }
        public Boolean IsUp { get; set; }
        public DateTime Time { get; set; }
    }
}
